### PR TITLE
Update arch.md

### PR DESCRIPTION
**Pull request by RileyMeta**
- Added additional testing information to line 3
- Added installation instructions for yay and the AUR along with a brief description of both
- Moved launch instructions to the bottom
- Changed launch instructions to ## instead of ### so it could be better distinguished

### CODE REVIEW

**Comment by dhtseany** [comment]
Hi! I'm Sean (dhtseany), if you wanted to note it I test all of my work
using Gnome, so essentially support for Gnome is ongoing with each release.
Thanks!

**Comment by RileyMeta** [comment]
Y'know what. I'm not worrying about the formatting anymore, this is aggravating, lol

**Comment by dhtseany** [comment]
Ha, you're doing good work, keep at it!

File: docs/installation/arch.md [hunk 1]
```diff
@@ -50,7 +54,30 @@ Please make sure you have followed [Install dependencies](#install-dependencies)
...
+
+### Install with yay
+
+[yay is a Pacman wrapper and AUR helper](https://aur.archlinux.org/packages/yay) that will download and install all of the prerequisites for you, along with the StreamDeck UI app itself.
+
+``` bash
+yay -S streamdeck-ui
+```
+
+> For both yay and the AUR you can also install the [Development branch](https://aur.archlinux.org/packages/streamdeck-ui-develop) if you want to stay up to date with improvements.
```
Comment (4s3ti) [inline docs/installation/arch.md:78]: Guess this bit should be a bit more clear. 
Development doesn't really mean that its to stay up to date, lets call it bleeding edge. 

streamdeck-ui is Stable release, new features will only drop here after a bit on the development branch, and for now l am updating this one every month or every 2 months depending. 
streamdeck-ui-develop, is unstable and under development for those who want to assist with testing or use the latest and greatest even if it means the application might break.

Comment (RileyMeta) [reply docs/installation/arch.md]: Updated it to make a little more sense, hopefully that keeps it clear and concise.

**Pull request merged**